Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
+MAINTAINER David Manouchehri
```

Comment:
Would you mind making this one of the core team? (myself, @mossberg, @feliam, @yan)